.eleventy.js

```diff
@@ -12,6 +12,12 @@ https://github.com/fluid-project/handbook.floeproject.org/raw/main/LICENSE.md.
 
 "use strict";
 
+var GithubSlugger = require("github-slugger");
+var githubSlugify = function (string) {
+    var slugger = new GithubSlugger();
+    return slugger.slug(string);
+};
```

Comment:
Sorry I should have mentioned that you probably don't need to do this wrapping. We did it in Infusion-Docs to minimize some name changes as we moved from a different system to GitHub-slugger. You can probably pass slugger.slug directly below.

Reply:
This seems like a pretty straightforward unwrapping, however I'm running into much difficulty getting 11ty to run after making this change:

```bash
[11ty] > Having trouble rendering liquid template ./src/collections/approaches/en-CA/AudioContentAndLearning.md

`TemplateContentRenderError` was thrown
[11ty] > Cannot convert undefined or null to object

`TypeError` was thrown:
[11ty]     TypeError: Cannot convert undefined or null to object
        at hasOwnProperty (<anonymous>)
        at Object.BananaSlug.slug [as slugify] (...\handbook.floeproject.org\node_modules\github-slugger\index.js:26:14)
        at Array.<anonymous> (...\handbook.floeproject.org\node_modules\markdown-it-anchor\dist\markdownItAnchor.js:1:5125)
        at Core.process (...\handbook.floeproject.org\node_modules\markdown-it\lib\parser_core.js:51:13)
        at MarkdownIt.parse (...\handbook.floeproject.org\node_modules\markdown-it\lib\index.js:524:13)
        at MarkdownIt.render (...\handbook.floeproject.org\node_modules\markdown-it\lib\index.js:544:36)
        at ...\handbook.floeproject.org\node_modules\@11ty\eleventy\src\Engines\Markdown.js:74:38
        at processTicksAndRejections (node:internal/process/task_queues:96:5)
        at async Template._render (...\handbook.floeproject.org\node_modules\@11ty\eleventy\src\TemplateContent.js:421:22)
        at async Template.getTemplateMapContent (...\handbook.floeproject.org\node_modules\@11ty\eleventy\src\Template.js:1056:19)
[11ty] Wrote 0 files in 0.62 seconds (v1.0.0)
```

I poked around in the source for [`markdown-it`](https://github.com/markdown-it/markdown-it) and [`markdown-it-anchor`](https://github.com/valeriangalliat/markdown-it-anchor/blob/master/index.js#L3), but was not able to determine the cause yet. I'll give this another shot tomorrow and see whether I can make it work. I suspect it may be due to some subtlety with the module export setup that I haven't picked up on yet.